src/main/java/com/arpnetworking/clusteraggregator/ClusterStatusCache.java

```diff
@@ -109,13 +128,51 @@ public Receive createReceive() {
                         }
                     }
                 })
+                .match(ShardRegion.ClusterShardingStats.class, shardingStats -> {
+                    LOGGER.debug()
+                            .setMessage("Received shard statistics")
+                            .addData("regionCount", shardingStats.getRegions().size())
+                            .log();
+                    final Map<String, Long> shardsPerAddress = Maps.newHashMap();
+                    final Map<String, Long> actorsPerAddress = Maps.newHashMap();
+                    for (final Map.Entry<Address, ShardRegion.ShardRegionStats> entry : shardingStats.getRegions().entrySet()) {
+                        final String address = entry.getKey().hostPort();
+                        for (final Map.Entry<String, Object> stats : entry.getValue().getStats().entrySet()) {
+                            final long currentShardCount = shardsPerAddress.getOrDefault(address, 0L);
+                            shardsPerAddress.put(address, currentShardCount + 1);
+                            if (stats.getValue() instanceof Number) {
+                                final long currentActorCount = actorsPerAddress.getOrDefault(address, 0L);
+                                actorsPerAddress.put(
+                                        address,
+                                        ((Number) stats.getValue()).longValue() + currentActorCount);
+                            }
+                        }
+                    }
+                    for (final Map.Entry<String, Long> entry : shardsPerAddress.entrySet()) {
+                        try (Metrics metrics = _metricsFactory.create()) {
+                            final Long actorCount = actorsPerAddress.get(entry.getKey());
+                            metrics.addAnnotation("address", entry.getKey());
+                            metrics.setGauge("akka/shards", entry.getValue());
+                            metrics.setGauge("akka/actors", actorCount);
```

Comment:
I'd consider adding a "cluster" segment to these keys. `akka/cluster/shards` `akka/cluster/actors`, just because as-is it could be misinterpreted as all actors in the system instead of just the ones owned by the cluster sharding manager. Shards wouldn't be ambiguous but I figure for consistency it'd be done to both.

Reply:
Done.